src/webapp/main.py

```
@@ -0,0 +1,85 @@
+
+
+class PriceSchema(BaseModel):
+    ticker: str
+    timestamp: str
+    open: float
+    close: float
+    high: float
+    low: float
+    volume: int
+
+    class Config:
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)